File: java/src/estruturas/condicional/Dardo.java
package estruturas.condicional;

import java.util.Scanner;

/**
 * Problema "dardo"
 * No arremesso de dardo, o atleta tem três chances para lançar o dardo
 * à maior distância que conseguir. Você deve criar um programa para,
 * dadas as medidas das três tentativas de lançamento, informar qual
 * foi a maior.
 */
public class Dardo {
  /**
   * The entry point of application.
   *
   * @param args the input arguments
   */
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);

    System.out.println("Digite as três distâncias:");

    final double a = sc.nextDouble();
    final double b = sc.nextDouble();
    final double c = sc.nextDouble();

    double greatest = c;
    if (a >= b && a >= c) {
      greatest = a;
    } else if (b >= c) {
      greatest = b;
    }

    System.out.printf("MAIOR DISTÂNCIA = %.2f%n", greatest);

    sc.close();
  }
}

File: python/src/estruturas/condicionais/dardo.py
""" Problema "dardo"
No arremesso de dardo, o atleta tem três chances para lançar o dardo
à maior distância que conseguir. Você deve criar um programa para,
dadas as medidas das três tentativas de lançamento, informar qual
foi a maior.
"""


def dardo() -> None:
    print("Digite as três distâncias:")

    a = float(input())
    b = float(input())
    c = float(input())

    greatest = c
    if a >= b and a >= c:
        greatest = a
    elif b >= c:
        greatest = b

    print(f"MAIOR DISTÂNCIA = {greatest:.2f}")


if __name__ == "__main__":
    dardo()
